charts/jaeger/templates/allinone-agent-svc.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if .Values.allInOne.enabled -}}
+
```

Comment:
if you are not changing the files meaningfully, avoid introducing formatting changes.